client/my-sites/site-monitoring/components/site-logs-table/site-logs-table-row.tsx

```diff
@@ -62,9 +82,10 @@ function renderCell(
 	}
 
 	if ( ( column === 'date' || column === 'timestamp' ) && typeof value === 'string' ) {
-		return moment( value )
+		const formattedDate = moment( value )
 			.utcOffset( siteGmtOffset * 60 )
-			.format( 'll @ HH:mm:ss.SSS Z' );
+			.format( 'h:mm A [on] MMM D, YYYY' );
```

Comment:
`ll` is the localized date, as far as I know. Are you intentionally changing it to a fixed format?